src/fish/data.ts

```diff
@@ -0,0 +1,5265 @@
+export interface GameDataFish {
+  /** The fish id */
+  id: number
+
+  /** The fish name */
+  name: string
+
+  /** The unlock items for the fish */
+  rarity: number
+
+  location: string
+
+  openWater: boolean
+
+  fishingHole: string[]
+
+  favoredBait: string
+
+  timeOfDay: string[]
+
+  fishingPower: number
+
+  achievement: {
+    achievementId: number
+    achievementBit: number
```

Comment:
Can probably call this just `id` and `bit`, but if you prefer this it's fine as well.